Governance/publish/publish-esgf.md

```diff
@@ -0,0 +1,5 @@
+# ESGF publishing procedure 
```

Comment:
@chloemackallah to populate these once merged I assume?

Reply:
Yes and Avi can work on the BoM part I'm waiting on the merge before adding him.